llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -3708,7 +3707,7 @@ void LoopVectorizationCostModel::collectLoopUniforms(ElementCount VF) {
 
     // Determine if all users of the induction variable update instruction are
     // uniform after vectorization.
-    auto UniformIndUpdate =
+    bool UniformIndUpdate =
         llvm::all_of(IndUpdate->users(), [&](User *U) -> bool {
```

Comment:
while you are editing around here, might as well drop redundant llvm::

```suggestion
       all_of(IndUpdate->users(), [&](User *U) -> bool {
```